sky/packages/sky/lib/widgets/transitions.dart

```diff
@@ -0,0 +1,169 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:sky/widgets/animated_component.dart';
+import 'package:sky/animation/animation_performance.dart';
+import 'package:sky/animation/animated_value.dart';
+import 'package:sky/widgets/animated_component.dart';
+import 'package:sky/widgets/basic.dart';
+import 'package:vector_math/vector_math.dart';
+
+abstract class TransitionBase extends AnimatedComponent {
+  TransitionBase({
+    Key key,
+    this.content,
```

Comment:
s/content/child/

We've tried to use the name `child` consistently when a widget has a unique widget child.  (This comment applies to the other uses of the term `content` in this patch.)